curriculum/challenges/english/14-responsive-web-design-22/learn-css-animation-by-building-a-ferris-wheel/step-014.md

```diff
@@ -11,22 +11,22 @@ Time to position the cabins around the wheel. Select the first `.cabin` element.
 
 # --hints--
 
-You should have a `.cabin:nth-of-type(1)` selector.
+You should have a `.cabin:nth-of-type(1) or .cabin:first-of-type` selector.
```

Comment:
I don't think we specifically need to mention the selector in the steps.

The challenge should still ask for `nth-of-type`, but allow `first-of-type` to pass.

Reply:
Sounds good, will push another edit to remove the first-of-type selector from hints.